### PR TITLE
added blog "Configure Mailazy as External SMTP Email Provider"

### PR DESCRIPTION
**Pull request by kunal-saini**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

##### Description of change

title: add blog "Configure Mailazy as External SMTP Email Provider"

description: blog explains steps and configurations to set in order to configure LoginRadius to send emails with Mailazy

##### Author Details for Blog Post Entry

id: Kunal
bio: Kunal is a Lead Software Development Engineer at Mailazy. He graduated in Computer Science and considers himself a learner of life. He loves to play cricket, football, and computer games.
github: https://github.com/kunal-saini

### CODE REVIEW

**Comment by kunal-saini** [comment]
Hey @raghunath-r-a @pareek-narendra @puneetsingh24 
Just wanted to check on the review, Is there something that needs to be changed. let me know. 

Thanks!

File: content/guest-post/configure-mailazy-as-external-smtp-email-provider/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**Step 3:** Enable and configure Mailazy as your email provider.
+
+Navigate to LoginRadius [Dashboard > Click Select & Configure > Configuration](https://dashboard.loginradius.com/configuration).
+
+![lr-dashboard](lr-dashboard.png)
+
+![lr-dashboard-app](lr-dashboard-app.png)
+
+**Step 4:** Expand "Email/SMS Provider Settings"
```
Comment (Akash-Patil) [inline content/guest-post/configure-mailazy-as-external-smtp-email-provider/index.md:34]: The steps under `Step 4: Expand "Email/SMS Provider Settings"` have to be updated to match the current flow of adding a SMTP provider in LoginRadius Developer account. Kindly update this.

File: content/guest-post/configure-mailazy-as-external-smtp-email-provider/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+![lr-dashboard-app](lr-dashboard-app.png)
+
+**Step 4:** Expand "Email/SMS Provider Settings" 
+
+1. Select SMTP Provider Settings and select `other` in the "SMTP Provider" dropdown, type in "From Name" and "From Address".
+2. Type "SMTP Host" value as `smtp.mailazy.com` and "SMTP Port" value as `587`.
+3. Paste generated API Key in "SMTP User Name" and API Secret in "SMTP Password".
+4. Enable SSL.
+
+![lr-smtp](lr-smtp.png)
```
Comment (Akash-Patil) [inline content/guest-post/configure-mailazy-as-external-smtp-email-provider/index.md:41]: This screenshot seems to be outdated. Please add a screenshot with updated UI.

**Review by Akash-Patil — CHANGES_REQUESTED** [review]
Hi @kunal-saini ,

I have reviewed the document and suggested a few changes. Please have a look.

Regards,
@Akash-Patil

**Comment by kunal-saini** [comment]
hey @raghunath-r-a @Akash-Patil since the LoginRadius developer plan no longer has the capability to configure SMTP, I would re-write the article for the enterprise plan/use-case. I have converted the PR to draft once done I will mark it ready for review again. 
thanks! 
cc; @kartiknc

**Comment by Akash-Patil** [comment]
> hey @raghunath-r-a @Akash-Patil since the LoginRadius developer plan no longer has the capability to configure SMTP, I would re-write the article for the enterprise plan/use-case. I have converted the PR to draft once done I will mark it ready for review again.
> thanks!
> cc; @kartiknc

Sure, No problem. 
Thanks.

**Comment by raghunath-r-a** [comment]
Closing this PR for now. Please create a new PR with updates as per new guidelines if needed. Thank you.

**Pull request closed without merging**